ui/lib/core/addon/helpers/engines-options-for-backend.js

```diff
@@ -0,0 +1,57 @@
+import { helper as buildHelper } from '@ember/component/helper';
```

Comment:
duplicated the concept of this helper, but using just for PKI engine at this point. Figured in the future, we will add more so I wanted to keep this pattern. Additionally, this way I don't touch the current PKI setup in the `options-for-backend.js` helper in the non-addon helper app folder